attributes/ctl.rb

```diff
@@ -1 +0,0 @@
-default['sensu-go']['ctl_version'] = '6.1.0'
```

Comment:
lets double check the readme to ensure nothing is referring to any of these removed attributes anymore.

Reply:
This can be removed it has been removed from the resource.